File: multilevel_inheritance.py
class MyClass:
    def my_function(self):
        print("Some content in parent class.")

# Child class
class MyChild(MyClass):
    pass

# Grandchild class
class MyGrandChild(MyChild):
    pass

if __name__ == "__main__":
    my_obj = MyGrandChild()
    my_obj.my_function()

File: multilevel_inheritance.cpp
#include<bits/stdc++.h>
#define fo(i,s,n) for(int i=s;i<n;i++)
#define ffo(i,s,n) for(int i=s;i<=n;i++)
#define ll long long int
#define ROBOT_RIAD ios_base::sync_with_stdio(false);cin.tie(0);cout.tie(0);
using namespace std;
typedef pair<int,int> pii;

// Parent class
class MyClass{
  public: 
    void myFunction() {
      cout << "Some content in parent class." ;
    }
};

// Child class
class MyChild: public MyClass {
};

// Grandchild class 
class MyGrandChild: public MyChild {
};

int main() {
    ROBOT_RIAD;
    MyGrandChild myObj;
    myObj.myFunction();
    return 0;
}
